jenkins-integration/jenkins-jobs/scenarios/oss-puppetserver-latest-jruby9k-json-vs-pson/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+node {
+    checkout scm
+    pipeline = load 'jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy'
+}
+
+pipeline.multipass_pipeline([
+        [
+                job_name: 'oss-latest-jruby-9k-json',
+                gatling_simulation_config: '../simulation-runner/config/scenarios/foss5x-medium-1250-2-hours.json',
+                server_version: [
+                        type: "oss",
+                        version: "latest"
+                ],
+                agent_version: [
+                        version: "latest"
+                ],
+                code_deploy: [
+                        type: "r10k",
+                        control_repo: "git@github.com:puppetlabs/puppetlabs-puppetserver_perf_control.git",
+                        basedir: "/etc/puppetlabs/code/environments",
+                        environments: ["production"],
+                        hiera_config_source_file: "/etc/puppetlabs/code/environments/production/root_files/hiera.yaml"
+                ],
+                background_scripts: [
+                        "./jenkins-jobs/common/scripts/background/curl-server-metrics-loop.sh"
+                ],
+                archive_sut_files: [
+                        "/var/log/puppetlabs/puppetserver/metrics.json"
+                ],
+                jruby_jar: "/opt/puppetlabs/server/apps/puppetserver/jruby-9k.jar",
+        ],
+        [
+                job_name: 'oss-latest-jruby9k-pson',
+                gatling_simulation_config: '../simulation-runner/config/scenarios/foss25x-medium-1250-2-hours.json',
```

Comment:
MMm... all good, I don't think it's probably worth the effort of re-articulating a new naming scheme for the files. I guess I'm just learning the correlations of the naming scheme to the what axes we care to test.